first_order_model/config/paper_configs/personalization/new_dataset.yaml

```diff
@@ -0,0 +1,110 @@
+dataset_params:
```

Comment:
please move to a new folder under configs, have a clear description of what this is doing, and what its relevance relative to other similar folders are, and get rid of any unnecessary files.